mpas_analysis/shared/io/mpas_reader.py

```diff
@@ -239,7 +239,7 @@ def _parse_dataset_time(ds, inTimeVariableName, calendar,
 
         # this is an array of date strings like 'xtime'
         # convert to string
-        timeStrings = [''.join(xtime).strip() for xtime in timeVar.values]
+        timeStrings = [''.join(xtime.astype('U')) for xtime in timeVar.values]
```

Comment:
@jhkennedy, I don't think the `.strip()` should have been lost here.  The `xtime` variable typically has trailing white space.

Reply:
Ah, blast. That's why I was having problems in timekeeping.  :anguished:

I agree, this should be there still.

Reply:
This is the only thing I haven't fixed because I didn't want to edit your commit history.  Once it's fixed, I'll run a few more tests but everything looks good to me so far.